src/themes/default/lv_theme_default.c

```diff
@@ -791,9 +791,10 @@ static void theme_apply(lv_theme_t * th, lv_obj_t * obj)
     else if(lv_obj_check_type(obj, &lv_button_class)) {
 
 #if LV_USE_TABVIEW
-        lv_obj_t * parent = lv_obj_get_parent(obj);
-        if(parent && lv_obj_get_index(parent) == 0) { /*Tabview header*/
-            if(lv_obj_check_type(lv_obj_get_parent(parent), &lv_tabview_class)) {
+        lv_obj_t * tv_header = lv_obj_get_parent(obj);
+        lv_obj_t * tv = tv_header ? lv_obj_get_parent(tv_header) : NULL;
```

Comment:
```suggestion
        lv_obj_t * tv = lv_obj_get_parent(tv_header);
```

It's OK to pass NULL, you'll get NULL too.